src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1828,7 +1828,7 @@ protected CSharpSyntaxNode GetBindableParentNode(CSharpSyntaxNode node)
                     return null;
                 }
                 
-                throw new ArgumentException($"The parent of {nameof(node)} must not be null unless this is a speculative semantic model.", nameof(node));
+                throw new ArgumentException(string.Format(CodeAnalysisResources.ParentMustNotBeNullUnlessSpeculativeSemanticModel, nameof(node)), nameof(node));
```

Comment:
>CodeAnalysisResources.ParentMustNotBeNullUnlessSpeculativeSemanticModel [](start = 58, length = 71)

It doesn't look like this message is used in VB. Should we move it to C# resources?